src/utils/roleAssignment.ts

```diff
@@ -0,0 +1,44 @@
+import { UnifiedRoleAssignment } from "@microsoft/microsoft-graph-types";
+import request, { CliRequestOptions } from "../request.js";
+
+const graphResource = 'https://graph.microsoft.com';
```

Comment:
Since we're using Graph URL in just one place, let's move it to the URL where we use it